src/js/common/index.js

```diff
@@ -5,9 +5,6 @@
 // other such libraries in here. Most of the site does not use these legacy
 // frameworks and it belongs in a lower-level module.
 
-// Global site style.
-require('../../sass/style.scss');
```

Comment:
@akainic I'm not sure of the root cause for the double imports, but moving `style.scss` from `index.js` to `no-react-entry.js` looks fine—the `no-react-entry.js` requires `index.js` either way

https://github.com/department-of-veterans-affairs/vets-website/blob/fix-js/src/js/no-react-entry.js#L4